k8s/mongodb/mongo-ss.yaml

```diff
@@ -72,7 +72,7 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 768Mi
+            memory: 3.5G
```

Comment:
Two points:
1. I referred to the links given below, and none of them specify memory as `X.XG`. I think the only unit supported is Mi (maybe Gi) in Kubernetes. Please verify this.
https://kubernetes.io/docs/tasks/administer-cluster/cpu-memory-limit/#enforcing-limits-at-point-of-creation
https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#how-pods-with-resource-limits-are-run
https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-ram-container/

2. The section on what happens if one does not specify memory or cpu limits is that the pod can use as much resource as possible. So, I am thinking if it wouldn't be better to just remove this configuration, and then open a tocket to ensure that we put in scheduling constraints so that Mongo is not co-located with Nginx or BigchainDB.
https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-ram-container/#if-you-dont-specify-limits-or-requests